Corona-Warn-App/src/main/java/de/rki/coronawarnapp/environment/download/DownloadCDNModule.kt

```diff
@@ -23,9 +24,9 @@ class DownloadCDNModule {
     @Singleton
     @DownloadCDNServerUrl
     @Provides
-    fun provideDownloadServerUrl(): String {
-        val url = BuildConfig.DOWNLOAD_CDN_URL
-        if (!url.startsWith("https://")) throw IllegalStateException("Innvalid: $url")
+    fun provideDownloadServerUrl(environment: EnvironmentSetup): String {
+        val url = environment.downloadCdnUrl
+        requireValidUrl(url)
```

Comment:
Could be shorter:
`return requireValidUrl(url)`

Reply:
made fun requireValidUrl return url  to carry out this change